README.md

```diff
@@ -1,18 +1,18 @@
 # Apache Spark
 
-[![Jenkins Build](https://amplab.cs.berkeley.edu/jenkins/job/spark-master-test-sbt-hadoop-2.7/badge/icon)](https://amplab.cs.berkeley.edu/jenkins/job/spark-master-test-sbt-hadoop-2.7)
```

Comment:
the reason i moved this to later is because if you just open the readme.md file in a text editor, it's not very welcoming to see a gigantic pile of img links.

Reply:
looks alright